Detect os: Rosa, MRE and Detect browser MAUI WAP Browser and Detect new brands: Xtouch, Fonos, Conquest and Detect devices for existing brands: Bush, Cat, DEXP, Lava, Explay, Artel, Pixus, Acer, Ark, MAXVI, Videocon, MicroMax, Huawei, Asus, Realme, Aligator, Unihertz, NOA, Maxtron, Alcatel, Irbis, Lenovo, Oukitel, Invin, Black Fox, Xiaomi, Ulefone, BQ, Wiko, Vestel, Odys, Kivi, Multilaser, Blackview, Haier, Nomu, Vertex, Coolpad

sanchezzzhak opened the pull request.

sanchezzzhak committed:
feat(browser) detect browser MAUI WAP Browser
feat(os) detect os MRE
feat(device) detect brand Bush model: SPIRA D5 Dual Camera
feat(device) detect brand Cat model: B100
feat(device) detect brand DEXP model: Larus TV1
feat(device) detect brand Lava models: Discover 136, KKT Quadra, KKT40 Power Plus
feat(device) detect brand Explay model: Fin
feat(device) detect brand Artel models: V5, X4, F5, M5
feat(device) detect brand MAXVI models: P11, K10, X300, X500, X800
feat(device) detect brand Videocon model: V1585
feat(device) detect brand MicroMax models: X291, X246, X458, X312, X252

sanchezzzhak committed:
feat(device) detect new brand Fonos model: O1
feat(device) detect brand Huawei models: Y5p (DRA-LX9), Honor X10 Lite (DNN-LX9)
feat(device) detect brand Asus model: ROG Phone 3 (ZS661KS)
feat(device) detect brand Realme model: 7 (RMX2155)
feat(device) detect brand Aligator model: S5520
feat(device) detect brand Unihertz model: Atom XL
feat(device) detect brand NOA model: Hummer R (2019)
feat(device) detect brand Maxtron model: S9
feat(device) detect brand Alcatel models: 1S (2020) (5028D), 3L (2020) (5029D),
 1SE (2020) (5030D), 3X (2020) (5061K, 5061U)

sanchezzzhak committed:
feat(device) detect brand Irbis model: TZ711
feat(device) detect brand lenovo model: Vibe P1
feat(device) detect brand Oukitel models: WP5, WP7, WP6, WP8 Pro

sanchezzzhak committed:
feat(device) detect brand Huawei model: P smart (2020) (POT-LX1A)
feat(device) detect brand Pixus model: Sprint
feat(device) detect brand Xiaomi model: MiTV 4A (MiTV-MSSP0)
feat(device) detect brand Invin model: KM7
feat(device) detect brand Ark model: Wizard 2
feat(device) detect brand DEXP models: 5.0" AL350, 4.5" AL250
feat(device) detect brand Black Fox model: B8 Fox+ (BMM442S)
feat(device) detect brand Ulefone models: Armor 3W, Armor 3WT
feat(device) detect brand BQ models: Magic O (BQ-6424L), Aurora SE (BQ-5732L), Jeans (BQ-5519G)
feat(device) detect brand Alcatel models: 3C (2019) (5006D), 1S (5024F), 3X (2019) (5048I)
feat(device) detect brand Wiko models: View Go (W_P130)
feat(device) detect brand Vestel models: Venus E4

sanchezzzhak committed:
feat(os) detect os linux Rosa

issue #6427

sanchezzzhak commented on regexes/oss.yml:
```diff
@@ -160,6 +160,13 @@
   name: 'MTK / Nucleus'
   version: '$1'
 
+##########
+# MRE / MAUI Runtime Environment
+##########
+- regex: 'MRE/(\d+)\.(\d+).*;MAUI'
+  name: 'MRE'
+  version: '$1.$2'
```
`MRE` not sure about the correct name, MRE is more of an SDK (created in Mediatek)

sgiehl replied:
Not sure here either. Tried to find some useful information on that, but couldn't :shrug: Other parsing libraries are detecting it as `MRE`, so might be fine.

sanchezzzhak committed:
feat(device) detect new brand Conquest models: S8, S6, S10, S11, S12, F2, S16
feat(device) detect brand Odys model: Pyro 7 Plus 3G

sanchezzzhak committed:
test: fix

sanchezzzhak committed:
feat(device) detect new brand Xtouch model: X10
feat(device) detect brand Acer model: Iconia One 10 (B3-A50FHD)
feat(device) detect brand Oukitel model: WP5000
feat(device) detect brand Multilaser model: F Pro
feat(device) detect brand Kivi model: 50U600GR
feat(device) detect brand BQ model: Practic (6030G)
feat(device) detect brand Blackview model: BV4900
feat(device) detect brand Haier model: Titan T5
feat(device) detect brand Nomu model: V1600S
feat(device) detect brand Vertex models: Pro P300 4G, Pro P310 4G
feat(device) detect brand Coolpad models: Cool 1 (C106-9), Cool S1 (C105-E0)

sanchezzzhak committed:
feat(device) detect brand Nomi model: Iron-X (I5070)
feat(device) detect brand EvroMedia model: Play Pad Pro XL
feat(device) detect brand Prestigio model: MultiPhone 5500 Duo
feat(device) detect Pixus models: Blaze 10.1" 3G, Play Five
feat(device) detect brand Vestel models: 5000, 5000 Dual, 5000 2GB
feat(device) detect brand Qumo models: Quest 510, Quest 474, Quest 456, Quest 455, Quest 507, Quest 506

issue #6355

sanchezzzhak committed:
feat(device) detect brand Qumo models: Sirius 1001, Sirius 890
feat(device) detect brand RoverPad model: Sky 7.85" 3G
feat(device) detect brand Gigabyte model: GSmart Simba SX1
feat(device) detect brand Wexler model: TAB 10iQ 3G
feat(device) detect brand Oysters model: T7V 3G
feat(device) detect brand BB Mobile models: Techno 9.7" 3G, Techno 8.0" 3G

issue #6355

sanchezzzhak committed:
feat(device) detect brand Ramos model: MOS1
feat(device) detect brand Blu model: Studio XL
feat(device) detect brand Huawei model: P40 Pro (NEY-NX9)
feat(device) detect brand Wiko models: Highway Pure, Highway Star

sanchezzzhak committed:
test: fix
feat(device) detect brand Wiko model: Y50 (W-K130-TMV, W-K130-OPE, W-K130-EEA), Sunny 3 Plus (W-K200), Harry 2 (W-V600)
fix(device) rename model for brand Wiko: Y50 (W-K420-EEA) => Y70 (W-K420-EEA)

sgiehl commented on regexes/device/mobiles.yml:
```diff
@@ -14222,15 +14331,19 @@ Wiko:
+    - regex: 'W-K130-(?:TMV|OPE|EEA)'
       model: 'Y50'
-    - regex: 'W-K510-(TVM|EEA|OPE|SUN)'
+    - regex: 'W-K420-EEA'
```
According to a quick search `W-K420-EEA` is actually the `Y50` as it was defined before. Where did you have the information from that it's `Y70`?

sanchezzzhak replied:
I used the official website https://world.wikomobile.com/red.php?telephone=2725

sgiehl replied:
ok fine. Guess the search results on Google are a bit confusing than...

liviuconcioiu commented:
W-K420-EEA is indeed Y50, as @sgiehl said.

`Wiko,Y50,W-K420,W-K420-EEA`

@sanchezzzhak can you please double check? I've added the link with devices that Google Play uses.

[https://storage.googleapis.com/play_public/supported_devices.csv](https://storage.googleapis.com/play_public/supported_devices.csv)
